diff -urN showimg-0.9.4-1.org/showimg/jpgoptions.cpp showimg-0.9.4-1/showimg/jpgoptions.cpp
--- showimg-0.9.4-1.org/showimg/jpgoptions.cpp	2004-11-28 19:05:44.000000000 +0100
+++ showimg-0.9.4-1/showimg/jpgoptions.cpp	2005-06-09 00:16:56.896963544 +0200
@@ -82,7 +82,7 @@
     qualitySslider1->setMaxValue( 100 );
     qualitySslider1->setValue( 75 );
     qualitySslider1->setOrientation( QSlider::Horizontal );
-    qualitySslider1->setTickmarks( QSlider::Above );
+    qualitySslider1->setTickmarks( QSlider::Left );
     qualitySslider1->setTickInterval( 0 );
     layout6->addWidget( qualitySslider1 );
 
